openbook_posts/models.py

```diff
@@ -85,6 +86,16 @@ def create_post(cls, creator, circles_ids=None, community_name=None, image=None,
 
         return post
 
+    @classmethod
+    def update_post(cls, post_id, text=None):
+
+        post = Post.objects.get(id=post_id)
```

Comment:
you can add an `update` method in the `Post` class that does this.

You can still use this method or simply do it on the user model with

> post = Post.objects.get(id=post_id)
> post.update(text=text)

I believe I'm doing this for a couple items already